src/main/java/com/manardenza/service/UserService.java

```diff
@@ -0,0 +1,36 @@
+package com.manardenza.service;
+
+import com.manardenza.dao.UserDaoImpl;
+import com.manardenza.entity.User;
+import com.manardenza.login.CurrentUser;
+
+public class UserService {
+
+    private static UserService instance;
+    private UserDaoImpl userDao;
+
+    private UserService() {
+        this.userDao = UserDaoImpl.getInstance();
+    }
+
+    public static UserService getInstance() {
+        if (instance == null) {
+            instance = new UserService();
+        }
+        return instance;
+    }
+
+    public void loginUser(String firstName, String lastName) {
+        User user = UserDaoImpl.getInstance().getUserByName(firstName, lastName);
```

Comment:
Why do you call UserDaoImpl.getInstance()? You already have instance of UserDao stored as instance variable